app/views/users/_admin_change_username.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2 class="heading"><%= t(".page_heading") %></h2>
+<%= error_messages_for :user %>
+<p class="note"><%= t(".description") %></p>
+
+<ul class="navigation actions" role="navigation">
+  <li><%= link_to t(".navigation.manage_user"), admin_user_path(@user) %></li>
+</ul>
+
+<%= form_tag changed_username_user_path(@user) do %>
+  <dl>
+    <dt><%= t("users.change_username.current_user_name") %></dt>
+    <dd><p class="important informational"><%= @user.login %></p></dd>
+    <dt><%= label_tag :new_login, t("users.change_username.new_user_name") %></dt>
+    <dd>
+      <%= text_field_tag :new_login, "user#{@user.id}", readonly: true, autocomplete: "off" %>
```

Comment:
This should have an `aria-describedby` attribute referencing the `id` of the footnote: `<%= text_field_tag :new_login, "user#{@user.id}", readonly: true, autocomplete: "off", "aria-describedby": "new-login-field-description" %>`